src/index.js

```diff
@@ -55,10 +55,18 @@ export default class extends three.Sprite {
     canvas.width = Math.max(...lines.map(line => ctx.measureText(line).width));
```

Comment:
Thanks for the PR @ettavolt. I think these two lines doing the resize are duplicate and not needed since there's another resize happening after the font is set. If so, could you remove them?